templates/settings/admin.php

```diff
@@ -47,19 +47,19 @@
 			<a target="_blank" rel="noreferrer noopener" class="icon-info" title="<?php p($l->t('Open Documentation'));?>" href="https://github.com/matiasdelellis/facerecognition/wiki/Confidence"></a>
 		</p>
 		<p class="settings-ranged">
-			<label for="min-confidence-range"><?php p($l->t('Less minimum confidence'));?></label>
+			<label for="min-confidence-range"><?php p($l->t('Lower minimum confidence'));?></label>
 			<span><input type="range" id="min-confidence-range" min="0.0" max="1.1" value="0.99" step="0.01" class="ui-slider"></span>
-			<label for="min-confidence-range"><?php p($l->t('Highest minimum confidence'));?></label>
+			<label for="min-confidence-range"><?php p($l->t('Higher minimum confidence'));?></label>
 			<span id="min-confidence-value"class="span-highlighted">...</span>
 			<a id="restore-min-confidence" class="icon-align icon-history" style="display: none;" title="<?php p($l->t('Restore'));?>" href="#"></a>
 			<a id="save-min-confidence" class="icon-align icon-edit" style="display: none;" title="<?php p($l->t('Save'));?>" href="#"></a>
 		</p>
 		<br>
 		<h3>
-			<?php p($l->t('Configuration information'));?>
```

Comment:
Here should be more information, such as the version of pdlib, if CUDA it is enabled, etc. So, don't change it. :wink: